### PR TITLE
SLURM improvements, /tmp/tmp* management fixes

### PR DESCRIPTION
**Pull request by JacekHoleczek**

### CODE REVIEW

File: ganga/GangaCore/__init__.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def getLCGRootPath():
 # ------------------------------------------------
 # store Ganga version based on new git tag for this file
 _gangaVersion = '7.1.9'
-_development = True
+_development = False
```
Comment (mesmith75) [inline ganga/GangaCore/__init__.py:32]: Please don't change this.

**Comment by JacekHoleczek** [comment]
Sorry, I did not change it myself. I was modifying original files from the ganga-7.1.9 release and that's how they came.
Can you fix it or do you want me to do something about it?

File: ganga/GangaCore/__init__.py (hunk 1 above)
Comment (JacekHoleczek) [reply ganga/GangaCore/__init__.py:32]: Sorry, I did not change it myself. I was modifying original files from the ganga-7.1.9 release and that's how they came.
Can you fix it or do you want me to do something about it?

**Comment by JacekHoleczek** [comment]
Let me try to fix it myself with a new pull request ... https://github.com/ganga-devs/ganga/pull/1409

**Comment by egede** [comment]
Great to put in a pull request. Just remember to branch from the `develop` branch in your fork and make sure that you have merged in the latest changes from there before you create the pull request.

**Comment by JacekHoleczek** [comment]
I work with stable released versions which I try to get working. I modify them in-place. The easiest way for me would be to send you a standard patch file. Doing a pull request is additional work and as you can see rather error-prone.

**Comment by egede** [comment]
Hi,

Sending a patch file is OK as well, but it might take a bit longer to 
implement a change s we then haev to include it ourselves in a pull request.

Ulrik.

**Pull request closed without merging**